qf_lib/data_providers/preset_data_provider.py

```diff
@@ -127,9 +127,9 @@ def get_price(self, tickers: Union[Ticker, Sequence[Ticker]], fields: Union[Pric
         self._check_if_cached_data_available(specific_tickers, fields, start_date, end_date)
         data_array = self._data_bundle.loc[start_date:end_date, specific_tickers, fields]
 
```

Comment:
General question - do we have a good test covering the case where this didn't work? If so, let's add it in this PR

Reply:
I added 2 additional tests for historical_price with aggregations